src/helpers.ts

```diff
@@ -133,18 +132,18 @@ export async function createBlankImage(
   return await image.png().toBuffer();
 }
 
-export function helpers(options) {
-  function directory(path: string): string {
-    return path.substr(-1) === "/" ? path : `${path}/`;
+function relativeTo(base: string | undefined | null, path: string): string {
+  if (!base) {
+    return path;
   }
 
-  function relative(path: string, relativeToPath = false): string {
-    return url.resolve(
-      (!relativeToPath && options.path && directory(options.path)) || "",
-      path
-    );
-  }
+  const directory = base.substr(-1) === "/" ? base : `${base}/`;
+  const url = new URL(path, new URL(directory, "resolve://"));
+
+  return url.protocol === "resolve:" ? url.pathname : url.toString();
+}
```

Comment:
Why do not use `file:///`?

Reply:
It was taken from [the suggestion](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/url.d.ts#L173-L195).

`file://` will also change semantics. If used passes `"file:///put-here/"` in options, it will be stripped. OTOH with `resolve://` protocol is preserved.